src/main/kotlin/FetchBlockIdCommand.kt

```diff
@@ -0,0 +1,22 @@
+import kotlinx.cli.ExperimentalCli
+import kotlinx.cli.*
+import kotlinx.coroutines.runBlocking
+
+@OptIn(ExperimentalCli::class)
+class FetchBlockIdCommand(private val apiClient: MempoolClient = MempoolClient()) :
+    Subcommand("fetchBlockId", "Fetch block ID for a given block height") {
+
+    private val startHeight by option(ArgType.Int, shortName = "s", description = "Start Height").required()
+    var result: String = ""
+
+    override fun execute() = runBlocking {
+        val blockIdResult = apiClient.fetchFirstBlockId(startHeight)
+        result = if (blockIdResult.isSuccess) {
+                        blockIdResult.getOrNull().toString()
+                    }
+                else {
+                        "Error fetching block ID: ${blockIdResult.exceptionOrNull()?.message}"
+                    }
+        println(result)
```

Comment:
nit: formatting is bad, you can run `Ctrl+Alt+L` to let IntelliJ format the file for you. You should always run that command on all your code to make sure it's properly formatted.